lib/clickhousex/query.ex

```diff
@@ -109,11 +110,26 @@ defimpl DBConnection.Query, for: Clickhousex.Query do
 
   defp query_type(statement) do
     cond do
-      Regex.match?(@create_query_regex, statement) -> :create
-      Regex.match?(@insert_query_regex, statement) -> :insert
-      Regex.match?(@select_query_regex, statement) -> :select
-      Regex.match?(@alter_query_regex, statement) -> :alter
-      true -> :update
+      statement
+      |> String.trim_leading()
+      |> String.downcase()
+      |> String.starts_with?(String.downcase(@create_query_keyword)) ->
+        :create
```

Comment:
can we please move this logic to a new function for more readability/reusability?
EX: `starts_with?(statement, @create_query_prefix) -> :create`